core-image.html

```diff
@@ -179,12 +212,94 @@
        * @type number
        * @default null
        */
-      height: null
+      height: Number
+    },
+
+    configure: function() {
+      return {
+        src: '',
+        preventLoad: false,
+        preload: false,
+        loading: false,
+        loaded: false,
+        placeholder: null,
+        fade: false,
+        sizing: null,
+        position: 'center',
+        width: null,
+        height: null,
+      };
+    },
+
+    bind: {
+      width: 'widthChanged',
+      height: 'heightChanged',
+      src: 'srcChanged',
+      placeholderBackgroundUrl: 'placeholderBackgroundUrlChanged'
+    },
+
+    computed: {
+      imageClassName: 'computeImageClassName(sizing)',
+      placeholderClassName: 'computePlaceholderClassName(fade,loaded,preload)',
+      placeholderBackgroundUrl: 'computePlaceholderBackgroundUrl(preload,placeholder)',
+      requiresPreload: 'computeRequiresPreload(preload,loaded)',
+      canLoad: 'computeCanLoad(preventLoad,src)',
+
+      // TODO(cdata): Change this to a bind when we can declare multiple
+      // bind dependencies in 0.8
+      _observe1: 'transformChanged(sizing,position)',
+      _observe2: 'loadBehaviorChanged(canLoad,preload,loaded)',
+      _observe3: 'loadStateChanged(src,preload,loaded)',
+    },
+
+    ready: function() {
+      if (!this.hasAttribute('role')) {
+        this.setAttribute('role', 'img');
```

Comment:
Perhaps? I'm not sure how I feel about publishing native attributes / properties (because of the getters and setters Polymer creates). Is this generally considered safe to do?